docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+app:
+  build: .
+  volumes:
+    - ./tock:/tock
+  links:
+    - db
+  working_dir: /tock
+  entrypoint: python /tock/docker_entrypoint.py
+  environment:
+    - PYTHONUNBUFFERED=yup
+    - DATABASE_URL=postgres://tock_user@db/tock
+    - RUNNING_IN_DOCKER=yup
+  command: python manage.py runserver 0.0.0.0:8000
+db:
+  image: postgres:9.4
+  environment:
+    - POSTGRES_DB=tock
+    - POSTGRES_USER=tock_user
+web:
+  image: nginx
+  volumes:
+    - ./provision/dev/nginx_site_docker.conf:/etc/nginx/conf.d/nginx_site_docker.conf
+  links:
+    - app
+  ports:
+    - "8000:80"
+sass:
+  build: .
```

Comment:
Note that I could've instead used the official [ruby Docker Hub image](https://hub.docker.com/_/ruby/) but then we would've had to create a separate ruby equivalent of `docker_entrypoint.py` so that SASS wouldn't create all the compiled CSS with root ownership. So it was easier to just add ruby+sass to the app container image, and reuse that.